composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=7.1.0"
```

Comment:
Why not `^7.0`? We cannot use `void` anyhow without releasing it as a new BC major...

Reply:
Oh, I see that we would need it for `iterable`. IMHO it's a good trade off then, since a great number of libs jumped directly from 5 to `^7.1` due to `void`.